vaadin-avatar-flow-parent/vaadin-avatar-flow/src/main/java/com/vaadin/flow/component/avatar/Avatar.java

```diff
@@ -295,6 +295,28 @@ public void setColorIndex(Integer colorIndex) {
         getElement().setProperty("colorIndex", colorIndex);
     }
 
+    /**
+     * Gets the enabled state of the avatar tooltip, which is {@code false} by
+     * default.
+     *
+     * @return <code>true</code> if the tooltip is shown on hover or focus,
+     *         <code>false</code> otherwise
+     */
+    public boolean isTooltipEnabled() {
```

Comment:
nit: why isn't the naming the same on the Flow & Web Components:
Web: "withTooltip"
Flow: "tooltipEnabled"

Reply:
The API `setTooltipEnabled()` was requested in https://github.com/vaadin/web-components/issues/1606
Also it's likely that Flow users will look for `setTooltip` in the autocomplete.

As for the web component, I don't have a strong opinion. We could also use `tooltip-enabled`.
The current `withTooltip` property name was inspired by `withBackdrop` in the overlay.

Generally, we don't have consistent boolean property names in different web components 🤷‍♂️ 
E.g. we have `clearButtonVisible` and `revealButtonHidden`, but also `withBackdrop`.

Reply:
Yeah, there definitely are some goof-ups in the current names of some properties. No changes are needed in the scope of this PR now anyways.